### ISSUE TITLE
Problem: integration documentation is not updated to "338-2"

### ISSUE THREAD

**Issue by lezzokafka** [crypto-org-chain/cronos-docs#43]
Following #38 :
Updated endpoints would be:
```
[RPCs for Cronos Testnet]
Tendermint: https://cronos-testnet-2.crypto.org or https://cronos-testnet-2.crypto.org:26657/
Cosmos RPC: https://cronos-testnet-2.crypto.org:1317/
gRPC: https://cronos-testnet-2.crypto.org:9090/
EVM HTTP JSON RPC: https://cronos-testnet-2.crypto.org:8545/
EVM WS JSON RPC: https://cronos-testnet-2.crypto.org:8546/
```

**Comment by aw126** [comment]
Updated in https://github.com/crypto-org-chain/cronos-docs/pull/46

**Comment by lezzokafka** [comment]
Closed by #46
